app/src/androidTest/java/org/wikipedia/Utils.kt

```diff
@@ -0,0 +1,59 @@
+package org.wikipedia
+
+import android.view.View
+import android.widget.TextView
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.ViewInteraction
+
+import androidx.test.espresso.matcher.BoundedMatcher
+import androidx.test.espresso.matcher.ViewMatchers.isAssignableFrom
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+
+
+class Utils {
+companion object {
+    fun atPosition(position: Int, @NonNull itemMatcher: Matcher<View?>): Matcher<View?>? {
+        checkNotNull(itemMatcher)
+        return object : BoundedMatcher<View?, RecyclerView>(RecyclerView::class.java) {
+            override fun describeTo(description: Description) {
+                description.appendText("has item at position $position: ")
+                itemMatcher.describeTo(description)
+            }
+
+            override fun matchesSafely(view: RecyclerView): Boolean {
+                val viewHolder = view.findViewHolderForAdapterPosition(position)
+                        ?: // has no item on such position
+                        return false
+                return itemMatcher.matches(viewHolder.itemView)
+            }
+        }
+    }
+
+
+    fun getTextOf(matcher: ViewInteraction): String {
+        var text = String()
```

Comment:
You can use more cleaner, kotlin approach for that: `var text: String`

Reply:
when I did that I got the message that "variable text must be initialized" 🤔